### ISSUE TITLE
🐛 BUG: empty dotfiles in public directory not moved to dist directory

### ISSUE THREAD

**Issue by Bloodyaugust** [snowpackjs/astro#689]
### What package manager are you using?

npm/yarn

### What operating system are you using?

Windows/Ubuntu/MacOSX

### Describe the Bug

Empty dotfiles such as `.nojekyll` (required for Github Pages hosting due to `_`-prepended directories in dist) are not moved from the public directory as part of `astro build`.

### Steps to Reproduce

1. have an astro project
2. add an empty dotfile like `.nojekyll` to `public/`
3. `astro build`
5. Empty dotfile is not moved to `dist/` directory

### Link to Minimal Reproducible Example (Optional)

https://github.com/bloodyaugust/greyson-site

**Comment by natemoo-re** [comment]
Maybe @drwpow can chime in here, didn't we land a fix for this upstream in Snowpack? If it's a `snowpack.config.mjs` setting, we probably enable it by default for Astro.

**Comment by drwpow** [comment]
Yes in order to maintain backwards-compatible behavior, including dotfiles in your final build is opt-in via the `mount.dot` option ([docs](https://www.snowpack.dev/reference/configuration#mount))

**Comment by natemoo-re** [comment]
> Yes in order to maintain backwards-compatible behavior, including dotfiles in your final build is opt-in via the `mount.dot` option ([docs](https://www.snowpack.dev/reference/configuration#mount))

Cool, thanks for the info! I would be `+1` on having Astro set `mount.dot` for `public` by default.

**Comment by Bloodyaugust** [comment]
Sounds great to me as well!

FWIW, setting `mount.dot` to `true` in a fresh `snowpack.config.js` in my project is throwing an error:
```
[11:57:42] [snowpack] ! /Users/grichey/Code/greyson-site/snowpack.config.js
Configuration Error:
  - snowpack.mount./Users/grichey/Code/greyson-site/dot is not exactly one from [subschema 0],[subschema 1]
[11:57:42] [snowpack] See https://www.snowpack.dev for more info.
(node:62795) UnhandledPromiseRejectionWarning: Error: ! /Users/grichey/Code/greyson-site/snowpack.config.js
Configuration Error:
  - snowpack.mount./Users/grichey/Code/greyson-site/dot is not exactly one from [subschema 0],[subschema 1]
    at handleValidationErrors (/Users/grichey/Code/greyson-site/node_modules/snowpack/lib/index.js:37050:11)
    at loadConfiguration (/Users/grichey/Code/greyson-site/node_modules/snowpack/lib/index.js:37329:13)
    at createSnowpack (file:///Users/grichey/Code/greyson-site/node_modules/astro/src/runtime.ts:387:26)
    at createRuntime (file:///Users/grichey/Code/greyson-site/node_modules/astro/src/runtime.ts:456:7)
    at build (file:///Users/grichey/Code/greyson-site/node_modules/astro/src/build.ts:61:19)
    at buildAndExit (file:///Users/grichey/Code/greyson-site/node_modules/astro/src/cli.ts:16:15)
```

**Comment by FredKSchott** [comment]
`mount.dot` was actually added specifically for this! I never went back and added it to Astro, but it sounds like there may be an error in Snowpack here? Would love for someone to help debug this if interested!

**Comment by jasikpark** [comment]
This would definitely be helpful since we're having a couple of users try deploying to Github Pages which requires a `.nojekyll` file. It'd be much easier if they could just add that to their `public/` folder rather than manually create it on deploy

**Comment by itday** [comment]
@Bloodyaugust I get the same error with this syntax:
```
module.exports = {
  mount: {
    public: '/dist',
    dot: true
  }
};
```
I checked the [snowpack.config.js](https://www.snowpack.dev/reference/configuration#mount) page and I believe the syntax is slightly different:
```
module.exports = {
  mount: {
    public: {url: '/dist', dot:true},
  }
};
```
The build process doesn't throw any errors now, but it doesn't seem to have any impact. The file is not copied.

**Comment by FredKSchott** [comment]
Thanks for the report! We believe that this has been fixed in astro v0.21, going out today. Let us know if you can still reproduce the issue after updating to v0.21 and we'll be happy to take a look.
